client/src/templates/Challenges/video/show.tsx

```diff
@@ -112,7 +112,7 @@ class ShowVideo extends Component<ShowVideoProps, ShowVideoState> {
       helpCategory
     });
     challengeMounted(challengeMeta.id);
-    this._container?.focus();
+    this._container?.current?.focus();
```

Comment:
This might be my fault, but `this._container` is never anything but `undefined`. So, the hotkeys do not work as expected - after pressing `n` or `p`, you have to focus on the main page body by clicking again, before you can type `n` or `p` again.

Go to any of the RDB challenges:
- Press `n`
- If nothing happens, press `Esc`, then `n` again
- Next lesson loads, press `n` again
- Nothing will happen because the focus is messed up

Reply:
pressing <kbd>n</kbd> and <kbd>p</kbd> doesn't do anything for me, even following your steps and allowing shortcuts in the settings.

But I think I get what you mean by focus is messed up, does the video below show that is fixed?

[rec-tab (2).webm](https://github.com/freeCodeCamp/freeCodeCamp/assets/88248797/fd46aec0-a8af-4942-b35d-ed2b0f91095b)

Reply:
The CodeAlly challenges are not video challenges... ?? Did you link the wrong video?

Reply:
> Go to any of the RDB challenges:

I am confused. Is the issue in Relation Database (RDB) challenges?

Reply:
Sorry, I confused myself.

I was talking about our pages: https://www.freecodecamp.org/learn/relational-database/learn-bash-by-building-a-boilerplate/build-a-boilerplate

- Go to: https://www.freecodecamp.org/learn/relational-database/learn-bash-by-building-a-boilerplate/build-a-boilerplate
- Press `n`
  - If nothing happens, press `Esc`
    - Press `n`
- On next page, press `n`, see nothing happen

Reply:
I think it should work as intended, except it's an editor page, because the focus with drop the first time on the editor, but after that it, it keeps focusing on the container